index.ts

```diff
@@ -106,30 +101,87 @@ wss.on('connection', function connection(ws) {
                 sendToClients("channelList", channels);
             }
             if (category === "newUser") {
-                const theUser = new User (message.username, message.usernum, 1, ++highestUserId, [0]);
+                const rows = await knex("serverid");
+                const sNum = rows[0].snum;
+                const theUser: User = {
+                    username: message.username,
+                    usernum: message.userNum,
```

Comment:
can you like always use `userNum` or `usernum`?

Reply:
userNum breaks with the sql but usernum looks worse imo, i'll migrate all of em to usernum